roles/socle-config/files/crd-conf-dso.yaml

```diff
@@ -194,6 +194,10 @@ spec:
                     chartVersion:
                       description: GitLab chart version (e.g., "6.11.10").
                       type: string
+                    pvcGitalySize:
+                      description: "Size for Gitaly, default: 50Gi"
```

Comment:
Je pense qu'on peut mettre `Size for Gitaly persistent volume`, on peut avoir la valeur par défaut en faisant `kubectl describe crd/dso-socle-configs.cloud-pi-native.fr`